jdaviz/app.py

```diff
@@ -72,32 +75,14 @@ class UnitConverterWithSpectral:
     def equivalent_units(self, data, cid, units):
         if cid.label == "flux":
             eqv = u.spectral_density(1 * u.m)  # Value does not matter here.
-            list_of_units = set(list(map(str, u.Unit(units).find_equivalent_units(
-                include_prefix_units=True, equivalencies=eqv)))
-                + [
-                    'Jy', 'mJy', 'uJy', 'MJy',
-                    'W / (Hz m2)', 'eV / (Hz s m2)',
-                    'erg / (Hz s cm2)', 'erg / (Angstrom s cm2)',
-                    'ph / (Angstrom s cm2)', 'ph / (Hz s cm2)',
-                    'ct'
-                ]
-                + [
-                    'Jy / sr', 'mJy / sr', 'uJy / sr', 'MJy / sr',
-                    'W / (Hz sr m2)', 'eV / (Hz s sr m2)',
-                    'erg / (s sr cm2)', 'erg / (Hz s sr cm2)',
-                    'erg / (Angstrom s sr cm2)',
-                    'ph / (Angstrom s sr cm2)', 'ph / (Hz s sr cm2)',
-                    'ct / sr'
-                ]
-                + [
-                    'Jy / pix2', 'mJy / pix2', 'uJy / pix2', 'MJy / pix2',
-                    'W / (Hz m2 pix2)', 'eV / (Hz s m2 pix2)',
-                    'erg / (s cm2 pix2)', 'erg / (Hz s cm2 pix2)',
-                    'erg / (Angstrom s cm2 pix2)',
-                    'ph / (Angstrom s cm2 pix2)', 'ph / (Hz s cm2 pix2)',
-                    'ct / pix2'
-                ]
+            all_flux_units = locally_defined_flux_units() + ['ct']
```

Comment:
is there a reason not to include `ct` (perhaps optionally) in the function?

Reply:
I wanted to keep it clear that 'locally_defined_flux_units' are units that can be converted to and from one another.

(This is a tangent but I wanted to write down my thoughts before I forget): I have some thoughts about how to make this clearer - locally_defined_flux_units is the list of units that appears in the dropdown if your loaded data is compatible with one of the units in that list. A more descriptive name might make sense, especially to make it clear that this list of units is the available conversions for data loaded in a compatible unit, and is not relevant otherwise.

If your loaded data is in counts, and you wanted to be able to convert from counts to e/s (not sure if this actually makes sense, just an example), then there would be another list of 'flux units' that is ['counts', 'e/s'] etc. 

Eventually we could use these lists of units we support and support conversions between to check the input against and raise warnings if you load a nonsense unit. We could also do away with the 'create_flux_equivalencies_list` function and just check if the input unit is compatible with one of these 'locally_defined_flux_units' lists, and append the input unit as a choice if it is (and if not, there will be no other choices for flux conversion so this will avoid the issue where it provides those units and the conversion breaks)

Reply:
Ok, thanks for the explanation.  The flux choices in unit conversion is still populated by `create_flux_equivalencies_list` - should that be changed here/now or not?

Reply:
Sounds good - deal!